joern-cli/frontends/rubysrc2cpg/src/main/scala/io/joern/rubysrc2cpg/astcreation/AstForExpressionsCreator.scala

```diff
@@ -27,12 +28,18 @@ trait AstForExpressionsCreator(implicit withSchemaValidation: ValidationMode) {
     case node: HashLiteral           => astForHashLiteral(node)
     case node: Association           => astForAssociation(node)
     case node: IfExpression          => astForIfExpression(node)
+    case node: RescueExpression      => astForRescueExpression(node)
     case _                           => astForUnknown(node)
 
   protected def astForStaticLiteral(node: StaticLiteral): Ast = {
     Ast(literalNode(node, code(node), node.typeFullName))
   }
 
+  
+  // Helper for nil literals to put in empty clauses
+  protected def astForNilLiteral: Ast = Ast(NewLiteral().code("nil").typeFullName(getBuiltInType(Defines.NilClass)))
+  protected def astForNilBlock: Ast = blockAst(NewBlock(), List(astForNilLiteral))
```

Comment:
There is a `blockNode(RubyNode)` method that uses the node info for code/location information. Maybe this can accept an `Option[RubyNode]` then you can `node.map(blockNode).getOrElse(NewBlock())`

Reply:
This takes inspiration from how the if expression is handled, as they insert a nil there. As long as dataflow can interpret an empty block as a nil value, then that logic is probably not necesary.